tests/compat.sh

```diff
@@ -534,6 +534,15 @@ add_mbedtls_ciphersuites()
     esac
 }
 
+# o_check_ciphersuite STANDARD_CIPHER_SUITE
```

Comment:
The input argument is a string with the format of "STANDARD CIPHER SUITE NAME = PROGRAM CIPHER SUITE NAME", how about changing it as?
```suggestion
# o_check_ciphersuite <"STANDARD_NAME=PROGRAM_NAME">
```

Reply:
I'd prefer not. The function is designed to be used as `o_check_ciphersuite STANDARD_CIPHER_SUITE`, but it also works if you pass `STANDARD_CIPHER_NAME=PROGRAM_CIPHER_NAME`. I'd rather change the usage (line 1050) to pass only the standard names.